janus_core/helpers/mlip_calculators.py

```diff
@@ -224,6 +224,21 @@ def choose_calculator(
         kwargs.setdefault("sevennet_config", None)
         calculator = SevenNetCalculator(model=model_path, device=device, **kwargs)
 
+    elif arch == "dpa3":
+        from deepmd import __version__
+        from deepmd.calculator import DP
+
+        if isinstance(model_path, Path):
+            model_path = str(model_path)
+        else:
+            raise ValueError(
```

Comment:
There's not necessarily a perfect way to handle this, but via the command line, something like 

```
janus singlepoint --arch dpa3 --struct tests/data/NaCl.cif --model-path ./invalid/path
```

gives raises error (the CLI input is a string, and this module only converts it to a `Path` if the path exists), which is misleading, as it makes it sound like `model_path` hasn't been set.

Perhaps we do want our own `Path.exists()` check, or just modify the error message to include making sure the path exists?